### ISSUE TITLE
Synthesizable

### ISSUE THREAD

**Issue by Soham-coder** [pulp-platform/fpnew#40]
Can this code be synthesized as it is in Synopsys DC.If not does it needs to be changed to synthesizable constructs? Thanks in advance.

**Comment by stmach** [comment]
Yes, the code can be synthesized as-is in Synopsys DC. For sure DC version `2019.03` and above are supported.

**Comment by Soham-coder** [comment]
Thanks a lot. Great work!
